### ISSUE TITLE
Does Depth_stream_examples support iPhone Xs AvDepthData

### ISSUE THREAD

**Issue by interactivetech** [huningxin/depth_stream_examples#2]
Hi there,

Great work! This is more of a question than a bug report. This is an exciting project and I was wondering if you happen to know if WebRTC or this project supports accessing the dual cameras on an iphone Xs?

**Comment by huningxin** [comment]
Hi @interactivetech , thanks for your comments. I'd suggest you to check out https://github.com/intel/depth-camera-web-demo. CC @riju who is leading this effort.

**Comment by riju** [comment]
No. MediaStream Image capture (depth) is not implemented for MacOS and also iOS.

**Comment by riju** [comment]
@interactivetech : If you are intersted to use RealSense cameras on Chrome, here are a few more demos you can check out - https://riju.github.io/depth-camera-web-demo/
